packages/cardpay-sdk/sdk/prepaid-card/base.ts

```diff
@@ -350,6 +350,7 @@ export default class PrepaidCard {
     faceValues: number[],
     customizationDID: string | undefined,
     onPrepaidCardsCreated?: (prepaidCardAddresses: string[], txnHash: string) => unknown,
+    onTxHash?: (txHash: string) => unknown,
```

Comment:
@luke the onPrepaidCardsCreated was supposed to fit this purpose--was there a problem with it?

Reply:
looking deeper I see that the logic to get the prepaid card ID's does actually require a txn receipt--which is why you'll needed to do this: 👍